src/app.tsx

```diff
@@ -17,7 +17,7 @@
  * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
  */
 
-import React, { useContext, useEffect, useState } from "react";
+import React, { useEffect, useState } from "react";
```

Comment:
For "move FilesContext and ClipboardInfo to avoid imports from app.jsx" -- This is rather unclean. This is half-fixup from the previous commit, and half-changing existing code. There should be a *first* commit that moves the existing app.tsx import (FolderFileInfo and useFilesContext) from the src/ components. This can even be its own PR and land quickly. Then you can squash the other half that was specific to the copy&paste.

Reply:
Ah, I see what you mean. Perhaps I misunderstood how to do this when we talked about it last week in person. I'll restructure the commit history (and hopefully not break anything)